.github/workflows/test-deploy.yml

```
@@ -89,45 +89,31 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ github.token }}
+      - name: Metadata
+        id: metadata
```

```suggestion
      - id: metadata
```
Maybe `id` is enough for consistency with other unnamed steps?

---

unfortunately the value of `run` gets displayed in the logs if there's no `name` and no `uses` :(